tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "target": "es2015",
+    "downlevelIteration": true,
+    "importHelpers": true,
+    "sourceMap": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "allowJs": false,
+    "checkJs": false,
+    "resolveJsonModule": true,
+    "lib": ["dom", "es2015", "es2016", "es2017", "esnext"],
+    "forceConsistentCasingInFileNames": true,
+    "experimentalDecorators": true,
+    "noImplicitAny": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noUnusedLocals": true,
+    "skipLibCheck": true,
+    "strictNullChecks": true,
```

Comment:
Add one `strict: true`, remove a bunch of other options
> --strict
>Enable all strict type checking options.
Enabling --strict enables --noImplicitAny, --noImplicitThis, --alwaysStrict, --strictBindCallApply, --strictNullChecks, --strictFunctionTypes and --strictPropertyInitialization

Reply:
THIS, PLEASE

Reply:
Should we do this in the other repos too @Velenir ? cause I think we only added --strictNullChecks explicitly at least

Reply:
Done f962b0af073b4da7d1ba7a7d3ee5d3c439f4e927